consumer/src/main/java/no/nav/sbl/dialogarena/modiabrukerdialog/consumer/service/oppgavebehandling/RestOppgaveBehandlingServiceImpl.kt

```diff
@@ -0,0 +1,447 @@
+                id = oppgaveId.toLong()
+        )
+        return response.toOppgaveJsonDTO()
+    }
+
+    private fun correlationId() = MDC.get(MDCConstants.MDC_CALL_ID) ?: UUID.randomUUID().toString()
+
+    private fun formatterBeskrivelseFerdigstiltOppgave(saksbehandlersValgteEnhet: String, gammelBeskrivelse: String?, beskrivelse: String) : String {
+        val beskrivelseFerdigstilt = "Oppgaven er ferdigstilt i Modia. " + beskrivelse
+        return leggTilBeskrivelse(gammelBeskrivelse, beskrivelseFerdigstilt, saksbehandlersValgteEnhet)
+    }
+
+
+
+    private fun getAktorId(fnr: String): String? {
+        return try {
+            pdlOppslagService
+                    .hentIdent(fnr)
+                    ?.identer
+                    ?.find { ident -> ident.gruppe == HentIdent.IdentGruppe.AKTORID }
+                    ?.ident
+                    // syntmapping for Q2 --> Q1
+                    ?.let(PdlSyntetiskMapper::mapAktorIdFraPdl)
+        } catch (e: Exception) {
+            null
+        }
+    }
+
+    private fun getFnr(aktorId: String): String? {
+        return try {
+            pdlOppslagService
+                    .hentIdent(aktorId)
+                    ?.identer
+                    ?.find { ident -> ident.gruppe == HentIdent.IdentGruppe.FOLKEREGISTERIDENT }
+                    ?.ident
+                    ?.let(PdlSyntetiskMapper::mapFnrTilPdl)
+        } catch (e: Exception) {
+            null
+        }
+    }
+
+    private fun oppgaveJsonDTOToOppgaveResponse(response: OppgaveJsonDTO): OppgaveResponse {
+        if (response.aktoerId == null || response.aktoerId!!.isEmpty()) {
+            throw Exception("AktørId mangler på person")
+        }
+        val fnr = getFnr(response.aktoerId!!)
+        if (fnr == null || fnr.isEmpty()) {
+            throw Exception("Fnr mangler på person")
+        }
+        val erSTO = Optional
+                .ofNullable(response.oppgavetype)
+                .map { kodeverksmapperService.mapOppgavetype(response.oppgavetype) }
+                .map { anObject: String? -> "SPM_OG_SVR" == anObject }
+                .orElse(false)
+        return OppgaveResponse(
+                response.id.toString(),
+                fnr,
+                response.journalpostId.toString(),
+                erSTO
+        )
+    }
+
+    private fun stripTemakode(prioritet: String): String {
+        return prioritet.substringBefore("_")
+    }
+
+    fun leggTilBeskrivelse(gammelBeskrivelse: String?, leggTil: String, valgtEnhet: String): String {
+        val ident = SubjectHandler.getIdent().orElseThrow { RuntimeException("Fant ikke ident") }
+        val header = String.format("--- %s %s (%s, %s) ---\n",
+                DateTimeFormat.forPattern("dd.MM.yyyy HH:mm").print(DateTime.now()),
+                ansattService.hentAnsattNavn(ident),
+                ident,
+                valgtEnhet)
+        val nyBeskrivelse = header + leggTil
```

Comment:
For stor fil. Kan du dele i 2 ellers 3 ?

Reply:
Har kommentaren noe med linje 444 å gjøre? Eller en generell kommentar? Enig i at vi kan prøve å trekke ut noe i utils filer osv